Ex9 html css fundamentals part1

Indigoinspn opened the pull request:
task.html – Created an Input form.
task-03.html – Created a button based on the layout.
task-04.html – Several approaches of positioning of an element.
task-08.html -  Grid and Flexbox layout.

Indigoinspn committed:
ex9_html-css-fundamentals-part1
task.html – Created an Input form.
task-03.html – Created a button based on the layout.
task-04.html – Several approaches of positioning of an element.
task-08.html -  Grid and Flexbox layout.

Indigoinspn committed:
add code to the file

aneelia commented:
стили надо вынести в отельные файлы

aneelia commented on src/ex9_html-css-fundamentals-part1/task-04.html:
```diff
@@ -0,0 +1,101 @@
+
+        /* approach 1 */
+
+        #b-1 {
```
названия - мрак. сделай что-то осмысленное, пожалуйста :) без бодер-1 2 3 и тд

Indigoinspn replied:
Изменила

aneelia commented on src/ex9_html-css-fundamentals-part1/task-08.html:
```diff
@@ -0,0 +1,171 @@
+
+        .grid section {
+            background: lightgreen;
+            float: left;
```
флоаты с гридами не работают

Indigoinspn replied:
Убрала

aneelia commented on src/ex9_html-css-fundamentals-part1/task-08.html:
```diff
@@ -0,0 +1,171 @@
+
+        .flex section {
+            background: lightgreen;
+            width: 75%;
```
когда верстаем на флексам, флоаты не используем и ширину желательно тоже. все можно сделать с использованием flex

Indigoinspn replied:
Поправила

aneelia commented on src/ex9_html-css-fundamentals-part1/task.html:
```diff
@@ -0,0 +1,183 @@
+
+        body {
+            font-family: 'Merriweather Sans', sans-serif;
+            margin: 5% 30%;
```
маржины в процентах не пишут, это такое себе

Indigoinspn replied:
Заменила

aneelia commented on src/ex9_html-css-fundamentals-part1/task.html:
```diff
@@ -0,0 +1,183 @@
+    <script defer>
+
+        function colorDark() {
+            document.getElementById('butt').style.backgroundColor = "#d0cbcb";
```
класс! умничка, что уже начала скрипты фигачить

Indigoinspn replied:
=)

Indigoinspn committed:
Put Css rules into separate files.
Task-04.html - corrected id names.
Task-08.html - remove floats and width properties.
Task.html -  change margin units.
